prog/vm/github_runner.rb

```diff
@@ -49,6 +49,8 @@ def self.assemble(installation, repository_name:, label:)
     end
   end
 
+  SERVICE_NAME = "runner-script.service"
```

Comment:
no need to add ".service", systemd-run already appends that. Currently your service name in OS would be runner-script.service.service

Reply:
I removed it